### DISCUSSION TITLE
issue: setting defaultValues for checkbox group items makes them non-changable

### DISCUSSION THREAD

**Discussion by PixeledCode** [react-hook-form/react-hook-form#10060] [Q&A]
### Version Number

7.43.5

### Codesandbox/Expo snack

https://codesandbox.io/s/wonderful-rui-t9qx50?file=/src/App.js

### Steps to reproduce

Checkbox Item `Cat` is checked; try to uncheck it.

### Expected behaviour

Checkbox Items should behave appropriately.
I have checked multiple versions of the package and the problem seems to start from `v7.22.5`, it's working fine on `v7.22.4`

### What browsers are you seeing the problem on?

Firefox

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by leapful** [comment]
@PixeledCode 

Latest version of react-hook-form will follow more strict HTML specification on value where `undefined` is not an acceptable value type and fallback to default values.

Please use empty string to solve the issue.

Working demo: https://codesandbox.io/s/dazzling-kapitsa-3jz928?file=/src/App.js
